terraform/dynamo.tf.json.template.py

```diff
@@ -23,6 +23,10 @@
                             {
                                 "name": "CartName",
                                 "type": "S"
+                            },
+                            {
+                                "name": "DefaultCart",
+                                "type": "N"  # Due to Terraform limitation, 0 or 1 will be used instead of boolean.
```

Comment:
Which limitiation is that, exactly?

Reply:
This is not a Terraform limitation, I guess, but a DynamoDB one. That's why I was confused.

```suggestion
                                "type": "N"  # Since DynamoDB lacks support for boolean values we represent them as 0 or 1
```

Reply:
It IS a DynamoDB restriction. See my reply to your comment.